docs/home/whats-new-2-15.md

```diff
@@ -1,94 +1,103 @@
-* Improved the readability of the node table to include a more detailed status and its description. For more information, see [Page here](page here)
+      * Improved views of the data
+      * Improved filters and search
+      * More information
 
-<!-- RUN-11421 Consumption report - Cost and bugs-->
-* Improved the Consumption report interface by moving the Cost settings to the *General* settings menu.
+    For more information see [Workloads](##workloads-view).
 
-<!-- RUN-5928 Node Scheduler - MS1 -->
+<!-- RUN-12313/12314 - CPU focused dashboards -->
+* Added a new dashboard for CPU based environments. The dashboards display specific information for CPU based nodes, node-pools, clusters, or tenants. These dashboards also include additional metrics that specific to CPU based environments. This will help optimize visual information eliminating the views of empty GPU dashlets. For more information see [CPU Dashboard](../admin/admin-ui-setup/dashboard-analysis.md#cpu-dashboard).
 
-<!-- RUN-7085/RUN-9480 Installation - Cluster wizard Improvements -->
+<!-- RUN-10622/RUN-10625 Policy blocks workloads that attempt to store data on the node-->
+* Added the ability to prevent the submission of workloads that use data sources that consists of a host path using policies. This is prevents data from being stored on the node. When a node is deleted, all data stored on that node is lost. For configuration information, see [Prevent Data Storage on the Node](##prevent-data-storage-on-the-node).
 
-<!-- RUN-7495/RUN11388 Support PSA / SCCs V2 \(security mechanism for pods on K8S/OCP\)-->
+--8<-- "home/whats-new-2-14.md:6:8"
 
-<!-- RUN-9386 Metrics API: Expose predefined metrics-->
-§
-<!-- RUN-9924/RUN-9925  Granular GPU compute time-slicing / Strict GPU compute time-slicing -->
+--8<-- "home/whats-new-2-14.md:15:16"
 
-<!-- RUN-9949 Container RUNtime Interface - No TW story as per ticket -->
+--8<-- "home/whats-new-2-14.md:18:20"
 
-<!-- RUN-9950 Support JFrog Artifactory private repository - No TW story as per ticket -->
+--8<-- "home/whats-new-2-14.md:26:27"
 
-<!-- RUN-10271/RUN-10321 Mark environment for workload type-->
+--8<-- "home/whats-new-2-14.md:29:31"
 
-<!-- RUN-10404/RUN-11747 Submit distributed training-->
+--8<-- "home/whats-new-2-14.md:33:35"
 
-<!-- RUN-10411/RUN-11390 Support self-signed certificates-->
+--8<-- "home/whats-new-2-14.md:45:47"
 
-<!-- RUN-10446/RUN-10447 RUN:ai Hyper-Scale - Scheduler - Phase 1: Adobe scenario-->
+## Improvements
 
-<!-- RUN-10451/RUN-10452 Support new Kubernetes and OpenShift releases - Q3/2023-->
+<!-- ADDLINK RUN-9943/RUN-12176 Nodes - reflect the correct status of the node - add to nodes page the table from the TW ticket -->
+* Improved the readability of the node table to include a more detailed status and its description. For more information, see [Node and Node Pool Status](##node-and-node-pool-status)
```

Comment:
Please add a sentence about the value in high level.
Something like - in order to easily assess any issue with the node that may impact resources availability in the cluster. Or something like that